### DISCUSSION TITLE
[next] Admin UI Field Groups and Dynamic Updates

### DISCUSSION THREAD

**Discussion by gautamsi** [keystonejs/keystone#5335] [Feature requests]
from https://keystone-next-docs-git-dim-placeholder-pages-in-docs.keystonejs.vercel.app/roadmap#admin-ui-field-groups-and-dynamic-updates

I see there is also a POC PR #4805 which brings Grouping feature in stable version. 

I have worked on a POC earlier #2260 which was for dynamically updating the UI based on the values. hopefully we will see them soon

**Comment by bladey** [comment]
Thanks for sharing this info @gautamsi.
